client/src/new-components/About.js

```diff
@@ -0,0 +1,93 @@
+import { ABOUT_TEXT } from '../constants';
+import styled from 'styled-components';
+
+const Image = styled.img`
+  width: 180px;
+  height: 180px;
+  object-fit: cover;
+  border-radius: 3px;
+  padding-bottom: 10px;
+`;
+
+const ContributorDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 20px;
+`;
+
+const ContribListDiv = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+`;
+
+const Header = styled.h1`
+  text-align: center;
+`;
+
+const Name = styled.span`
+  font-weight: bold;
+`;
+
+const School = styled.span`
+  font-size: 0.9rem;
+`;
+
+const Link = styled.a`
+  font-size: 0.9rem;
+`;
+
+const Text = styled.p`
+  padding: 10px;
+`;
+
+const About = () => {
+  const contributors = [
+    {
+      image: 'https://a1cf74336522e87f135f-2f21ace9a6cf0052456644b80fa06d4f.ssl.cf2.rackcdn.com/images/characters_opt/p-ses-bert.jpg',
+      name: 'Brandon Situ',
+      role: 'UI/UX, PM',
+      school: 'SFU Business',
+      linkedin: 'linkedin.com/in/brandonsitu',
+    }, {
+      image: 'https://smartcdn.prod.postmedia.digital/windsorstar/wp-content/uploads/2015/02/cookiemonster.jpg',
+      name: 'Jacob Pauls',
+      role: 'Full Stack',
+      school: 'BCIT Computing',
+      linkedin: 'linkedin.com/in/jacobpauls',
+    }, {
+      image: 'https://variety.com/wp-content/uploads/2020/05/not-too-late-show-with-elmo.jpg',
+      name: 'Siddharth Gupta',
+      role: 'Back End',
+      school: 'SFU Computer Science',
+      linkedin: 'linkedin.com/in/siddg97',
+    }, {
+      image: 'https://ichef.bbci.co.uk/images/ic/640x360/p01wf46x.jpg',
+      name: 'Shirley Vong',
+      role: 'Front End',
+      school: 'SFU Software Systems',
+      linkedin: 'linkedin.com/in/shirley-vong',
+    },
```

Comment:
in the `linkedIn` key you should add the `https://` prefix for the urls. Right now it thinks its a route in the react app and not an external link